arraycontext/loopy.py

```diff
@@ -98,22 +100,22 @@ def get(c_name, nargs, naxes):
 
         import loopy as lp
 
-        from .loopy import make_loopy_program
         from arraycontext.transform_metadata import ElementwiseMapKernelTag
+
+        def sub(name: str) -> Variable | Subscript:
+            return Variable(name)[subscript] if subscript else Variable(name)
```

Comment:
This is meant to silence the deprecation in `pymbolic` about an empty subscript. I went with keeping the same behavior here, i.e. returning the expression itself if the subscript is empty. Let me know if it looks ok!